lib/active_merchant/billing/gateways/braintree_blue.rb

```diff
@@ -412,7 +412,8 @@ def response_from_result(result)
           message_from_result(result),
           response_hash,
           authorization: result.transaction&.id,
-          test: test?
+          test: test?,
+          amount: result.transaction.amount * 100
```

Comment:
Just making it consistent between two calls

```suggestion
          amount: result.transaction.amount * 1_00
```

Reply:
refactored